docs/streams/stream-examples/realtime-updates.md

```diff
@@ -43,53 +212,49 @@ const sleep = (milliseconds) => {
   return new Promise(resolve => setTimeout(resolve, milliseconds));
 };
 
-// Create an authenticated instance with a token or API key
-
-// const client = new jsc8({url: globalUrl, token: "XXXX", fabricName: '_system'});
-// const client = new jsc8({url: globalUrl, apiKey: "XXXX", fabricName: '_system'});
-
-// Or use email and password to authenticate client instance
-
-async function createCollection () {
-  console.log("\n1. Log in.");
-  await client.login(email, password);
-  console.log("\n2. Create collection.");
-  try {
-    console.log(`Creating the collection ${collectionName}...`);
-    const existsColl = await client.hasCollection(collectionName);
-    if (existsColl === false) {
-      await client.createCollection(collectionName, { stream: true });
-    }
-    // Add an onChange listener for collection
-    listener = await client.onCollectionChange(collectionName);
-    
-    // Decode the message printed here in readable format
-    listener.on("message", (msg) => {
-      const receivedMsg = msg && JSON.parse(msg);
-      console.log("message=>", Buffer.from(receivedMsg.payload, "base64").toString("ascii"))
-    });
-    listener.on("open", () => console.log("Connection open"));
-    listener.on("close", () => console.log("Connection closed"));
-  } catch (e) {
-    await console.log("Collection creation did not succeed due to " + e);
-  }
-}
-async function insertData () {
-  console.log(`\n3. Insert data in region ${globalUrl}`);
-  await client.insertDocumentMany(collectionName, data);
+async function main () {
+	async function createCollection () {
+		console.log("\n1. Log in.");
+		console.log("\n2. Create collection.");
+		try {
+		  console.log(`Creating the collection ${collectionName}...`);
+		  const existsColl = await client.hasCollection(collectionName);
+		  if (existsColl === false) {
+			await client.createCollection(collectionName, { stream: true });
+		  }
+		  // Add an onChange listener for collection
+		  listener = await client.onCollectionChange(collectionName);
+
+		  // Decode the message printed here in readable format
+		  listener.on("message", (msg) => {
+			const receivedMsg = msg && JSON.parse(msg);
+			console.log("message=>", Buffer.from(receivedMsg.payload, "base64").toString("ascii"))
+		  });
+		  listener.on("open", () => console.log("Connection open"));
+		  listener.on("close", () => console.log("Connection closed"));
+		} catch (e) {
+		  await console.log("Collection creation did not succeed due to " + e);
+		}
+	}
+	await createCollection();
+
+	async function insertData () {
+		console.log(`\n3. Insert data`);
+		await client.insertDocumentMany(collectionName, data);
+	}
+	await sleep(2000);
+	await insertData();
+
+	async function deleteData () {
+		console.log("\n4. Delete data");
+		await client.deleteCollection(collectionName);
+	}
+	await sleep(10000);
+	// Closing OnChange listener
```

Comment:
Generally, I like to leave an empty line above comments. It helps with readability.

Reply:
Also, avoid gerunds in favor of active verbs, and don't forget to put periods at the end of sentences.